src/live.rs

```diff
@@ -782,6 +788,20 @@ RequiredBy=multi-user.target"#.into(),
             true)
     }
 
+    pub fn dest_device(&mut self, device: &str, overwrite: bool) -> Result<()> {
+        println!(
+            "Warning: boot media will overwrite {} without confirmation.",
+            device
+        );
+        if !overwrite {
+            bail!("specify --overwrite-dest-device to continue");
+        }
```

Comment:
Any reason for this? Our stance so far has been that you better know what you're doing re. passing the device name.

Reply:
Part of that stance has been that we won't autodetect the install device for you, because it's too dangerous; you have to specify it explicitly.  But here we're creating an image that overwrites the target disk without explicit user action, so I thought an additional safeguard would be appropriate.

However, it's reasonable to argue that we're time-shifting the explicit user action rather than eliminating it, and if the user wants a timebomb we should give it to them.  I'll drop the hidden option.